.cspell.json

```diff
@@ -177,6 +193,7 @@
         "txbytecode",
         "txhashoutputs",
         "txtime",
+        "verson",
```

Comment:
You sure you want to include this for a misspelling test?

Reply:
You're right, I just didn't know how to disable spellchecking on a single line, but I looked it up: `// cspell:disable-next-line`